.github/workflows/macos_x86_64.yml

```diff
@@ -1,33 +1,33 @@
 on:
-    workflow_call:
+  workflow_call:
 
 name: Macos x86-64 rust tests
 
 env:
-    RUST_BACKTRACE: 1
```

Comment:
I didn't intend to format these `.yml` files, I think zed must have done that and I hadn't noticed.